apps/admin/src/pages/sponsors/add-sponsor-form.tsx

```diff
@@ -53,9 +53,11 @@ export function AddSponsorForm() {
   });
 
   // Fill with valid hackathonId's
-  // Use hook directly within the component body or useEffect
   const { data: hackathons } = trpc.hackathons.getAll.useQuery();
 
+  // initialize tiers
+  const tiers = ["gold", "silver", "bronze"];
```

Comment:
This constant should probably be moved to `packages/db/src/consts` and then imported accordingly.

Reply:
Constants look like this: 
```ts
export const gradYears = [
  "2024",
  "2025",
  "2026",
  "2027",
  "2028",
  "other",
] as const;
```
Notice the `as const` which ensures this object is immutable.

Reply:
```ts

export const sponsors = sqliteTable("sponsors", {
  id: integer("id").primaryKey({ autoIncrement: true }),
  name: text("name").notNull(),
  logo: text("logo").notNull(),
  website: text("website").notNull(),
  tier: text("tier", { enum: ["gold", "silver", "bronze", "other"] }).notNull(),
  hackathonId: integer("hackathon_id").references(() => hackathons.id, {
    onDelete: "cascade", // If the hackathon is deleted, delete the hacker
    onUpdate: "cascade", // If the hackathon is updated, update the hacker
  }),
});
```

Another thing is in `packages/db/src/schemas/schema.ts`, which is where the above code snippet comes from should be updated to where the enum property should point to the tiers constant.

Reply:
What a marvelous convention. Implementing this fills me with excitement.